tests/unit/backend/chalice/api_server/test_v1_dataset.py

```diff
@@ -22,6 +24,7 @@ def test__post_dataset_asset__OK(self):
         self.assertIsNotNone(presign_url)
         self.assertEqual(expected_body, actual_body)
 
+    @unittest.skipIf(os.getenv("IS_DOCKER_DEV"), "This is currently TODO in docker")
```

Comment:
NOTE: This file will exist in the docker environment since localstack is stateful.